### ISSUE TITLE
WCAG 4.1.2: Ensures buttons have discernible text (.hamburger.icb[onclick="apz.toggleSidebar(this)"])

### ISSUE THREAD

**Issue by Report-Accesibility** [Report-Accesibility/Report#1]
#### Issue
`Ensures buttons have discernible text` ([button-name](https://accessibilityinsights.io/info-examples/web/button-name))
#### Target application
[Bank Application](https://apzautobankapp.apzplatforms.com/BankApp/)
#### Element path
#bfpapp__BasePage__sc_col_15 > .hamburger.icb[onclick="apz.toggleSidebar(this)"]
#### Snippet
`<button id="bfpapp__BasePage__mainMenu" type="button" onclick="apz.toggleSidebar(this)" class="ett-bttn pri med icb hamburger btn-icn" enabled="enabled">`
#### How to fix
    Fix any of the following:
      Element does not have inner text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
      Element's default semantics were not overridden with role="none" or role="presentation"
#### Environment
Chrome version 104.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.32.0 (axe-core 4.4.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
